### PR TITLE
Discord-Exam_Remind_Bot

### PR DESCRIPTION
**Pull request by kelvin0179**
## Pull Request Checklist

Go through the check boxes given below and make sure you mark the relevant ones.
#### To check put x inside [ ] eg. [x] Extension

Is your project an
- [ ] Extension 
- [x] Application

Given below are the mandatory requirements we expect from your project. Your Pull Request will be reviewed only if all the minimum requirements relevant to your project are satisfied.

- [x] Added readme.md to my project folder with relevant information.
- [x]  I have provided the demo video link (showing the functionality in action) or the public deployment link of my extension of application within the readme.md (if Applicable).
- [x] I have explained the process in a readable manner within readme.md.

### Terms and Conditions

`Plagiarism check from Team Crio will be solid and if your code is found to be plagiarized, the team has every right to reject your Pull Request and take necessary actions.`
- [x] I can assure you that this is my own contribution and I did not involve in any kind of plagiarism activities.

Buckle up and wait till you recieve communication from the review team. Keep a tab on this pull request page for comments on your project. Any necessary suggestions will also be provide here. All the best :)

### CODE REVIEW

**Review by theAdityaNVS — APPROVED** [review]
Good one @kelvin0179,
Useful bot for Exam Servers !!

**Comment by sudhanshutiwari264** [comment]
@kelvin0179 fantastic work!
A demo video is mandatory, so please include it!

File: Discord-Exam-Reminder/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+### The get command according to help.
+![5](https://user-images.githubusercontent.com/56430190/124385644-456f8700-dcf4-11eb-81a4-1038fc6053e8.PNG)
+### The delete command according to help.
+![6](https://user-images.githubusercontent.com/56430190/124385646-46081d80-dcf4-11eb-810b-594c671976b7.PNG)
+### The get after deleting.
+![7](https://user-images.githubusercontent.com/56430190/124385648-46081d80-dcf4-11eb-8e30-90592512e25c.PNG)
+### Alert on the upcoming quiz
+![image](https://user-images.githubusercontent.com/56430190/124427267-ca0be500-dd88-11eb-9fff-52ace1343cd4.png)
+
+
```
Comment (sudhanshutiwari264) [inline Discord-Exam-Reminder/README.md:118]: @kelvin0179 fantastic work!
A demo video is mandatory, so please include it!

**Review by sudhanshutiwari264 — CHANGES_REQUESTED** [review]
@kelvin0179 fantastic work!

File: Discord-Exam-Reminder/README.md (hunk 1 above)
Comment (kelvin0179) [reply Discord-Exam-Reminder/README.md:118]: @sudhanshutiwari264  There is a demo video at the top .
If that is not sufficient then I can create another one.

Comment (sudhanshutiwari264) [reply Discord-Exam-Reminder/README.md:118]: @kelvin0179 LGTM!
As it's on top, I hadn't noticed it!

**Review by sudhanshutiwari264 — APPROVED** [review]
great work!

File: Discord-Exam-Reminder/README.md (hunk 1 above)
Comment (kelvin0179) [reply Discord-Exam-Reminder/README.md:118]: @sudhanshutiwari264 Thanks a lot for the feedback!!

**Review by Nitesh-thapliyal — COMMENTED** [review]
Hey @kelvin0179 Nice work👏

Here is a suggestion that you can add to your bot:
- Does the bot contains a feature to change the prefix of command? If  no can you add so that the user can change the prefix of command as many bots are discord has similar commands prefix

**Comment by kelvin0179** [comment]
> Hey @kelvin0179 Nice work👏
> 
> Here is a suggestion that you can add to your bot:
> 
> * Does the bot contains a feature to change the prefix of command? If  no can you add so that the user can change the prefix of command as many bots are discord has similar commands prefix

**Comment by kelvin0179** [comment]
> Hey @kelvin0179 Nice work👏
> 
> Here is a suggestion that you can add to your bot:
> 
> * Does the bot contains a feature to change the prefix of command? If  no can you add so that the user can change the prefix of command as many bots are discord has similar commands prefix

@Nitesh-thapliyal  yes the bot does have a prefix before every command '$' 
although the users using the bot cannot edit it .
But it would be pretty cool if the users can edit the prefix

**Comment by Nitesh-thapliyal** [comment]
> > Hey @kelvin0179 Nice work👏
> > Here is a suggestion that you can add to your bot:
> > 
> > * Does the bot contains a feature to change the prefix of command? If  no can you add so that the user can change the prefix of command as many bots are discord has similar commands prefix
> 
> @Nitesh-thapliyal yes the bot does have a prefix before every command '$'
> although the users using the bot cannot edit it .
> But it would be pretty cool if the users can edit the prefix

yes can you add the feature so that user can change the prefix of command from the discord

**Comment by jainaayush01** [comment]
@kelvin0179 any updates?

**Comment by kelvin0179** [comment]
> @kelvin0179 any updates?

Its been 3 days and the automatic alerts and deletion is working fine

**Review by jainaayush01 — APPROVED** [review]
> yes can you add the feature so that user can change the prefix of command from the discord

this feature was suggested so we were waiting for that...

If you have implemented the feature let us know, or we will move forward and merge this PR

**Comment by kelvin0179** [comment]
> > yes can you add the feature so that user can change the prefix of command from the discord
> 
> this feature was suggested so we were waiting for that...
> 
> If you have implemented the feature let us know, or we will move forward and merge this PR

@jainaayush01 Really sorry about that , I have not implemented it , you can merge my request .

**Review by archithdwij — APPROVED** [review]
@kelvin0179 Try to update the prefix whenever time permits. Congratulations on getting your PR merged 🎆

**Comment by archithdwij** [comment]
@all-contributors Please add @kelvin0179 for code,idea,infra,design

**Comment by archithdwij** [comment]
@all-contributors Please add @kelvin0179 for code, idea, infra, design

**Pull request merged**